pkg/imgpkg/bundle/bundle.go

```diff
@@ -87,11 +107,7 @@ func (o *Bundle) findCachedImageRef(digestRef string) (ImageRef, bool) {
 }
 
 func (o *Bundle) allCachedImageRefs() []ImageRef {
```

Comment:
i think we should remove this accessor since in bunch of places we are using o.cachedImageRefs directly.

Reply:
done